taskcluster/kinds/extract-best/kind.yml

```diff
@@ -81,11 +82,13 @@ tasks:
                 - >-
                     zstd -d --rm $MOZ_FETCHES_DIR/*.zst &&
                     export PYTHONPATH=$PYTHONPATH:$VCS_PATH &&
+                    pip install --upgrade pip &&
+                    pip install -r $VCS_PATH/pipeline/translate/requirements/bestbleu.txt &&
                     python3 $VCS_PATH/pipeline/translate/bestbleu.py
```

Comment:
Suggestion: This file name doesn't really make sense anymore, since it's not using bleu, but chrf. Can we rename it to `extract_best.py` to have it match the kind?